src/shared/Form/Input.tsx

```diff
@@ -1,7 +1,14 @@
 import { inputStyles } from '../Styles/inputStyles';
+import { IInputProps } from '../Styles/inputStyles';
 import styled from '../Styles/styled-components';
 
-export const Input = styled.input`
+const inputBase = styled.input``;
+
+export const Input = styled(inputBase)<IInputProps>`
```

Comment:
Does this need a closing tag for IInputProps

Reply:
I think the `<IInputProps>` isn't a tag but it's the generic type parameter thing (not sure the exact name for it in TS)

Reply:
Ahh generics?

Reply:
Yes! After I updated typescript it wasn't happy with what was there before